src/providers/lambda/api.ts

```diff
@@ -0,0 +1,24 @@
+import { ProviderAPIConfig } from '../types';
+
+export const LambdaAPIConfig: ProviderAPIConfig = {
+  getBaseURL: () => `https://api.lambdalabs.com/v1`,
+  headers({ providerOptions }) {
+    const { apiKey } = providerOptions;
+    const headers = {
+      Authorization: apiKey,
```

Comment:
As per their docs examples, `Bearer` is required in the `Authorization` header. Can we please add it here as well?

Reply:
https://docs.lambdalabs.com/on-demand-cloud/using-the-lambda-chat-completions-api#creating-chat-completions